src/style.css

```diff
@@ -1,3 +1,78 @@
+html {
+  font-family: Roboto, sans-serif;
+  font-weight: 500;
+  font-size: 12px;
+}
+
+* {
+  margin: 0;
+  padding: 0;
+}
+
 body {
+  background-color: #858585;
+}
+
+.nav {
+  display: flex;
+  justify-content: flex-end;
+  font-size: 15px;
+  align-items: center;
+  height: 60px;
```

Comment:
It's not recommended to fix the container size if there is no such requirement. Let the content size dictate it to avoid overflow or accidental scroll bar.